File: CS/DevExpressMvcApplication1/Models/ResourcesSorting.cs
using System;
using System.Collections;
using System.Collections.Generic;
using System.Linq;
using System.Web;

public class ResourceSortType {
    public int ID { get; set; }
    public string Type { get; set; }
}

public enum ResourcesSortOrder {
    None = 0,
    Ascending = 1,
    Descending = 2,
    NOfAppointments = 3
}

public class ResourcesSortingHelper {
    public static IEnumerable GetItems() {
        return new List<ResourceSortType>() {
            new ResourceSortType() { ID = (int)ResourcesSortOrder.None, Type = "None"},
            new ResourceSortType() { ID = (int)ResourcesSortOrder.Ascending, Type = "A-Z Order"},
            new ResourceSortType() { ID = (int)ResourcesSortOrder.Descending, Type = "Z-A Order"},
            new ResourceSortType() { ID = (int)ResourcesSortOrder.NOfAppointments, Type = "N-Appointments"},
        } as IEnumerable;
    }
}

File: VB/DevExpressMvcApplication1/Models/ResourcesSorting.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web

Public Class ResourceSortType
	Private privateID As Integer
	Public Property ID() As Integer
		Get
			Return privateID
		End Get
		Set(ByVal value As Integer)
			privateID = value
		End Set
	End Property
	Private privateType As String
	Public Property Type() As String
		Get
			Return privateType
		End Get
		Set(ByVal value As String)
			privateType = value
		End Set
	End Property
End Class

Public Enum ResourcesSortOrder
	None = 0
	Ascending = 1
	Descending = 2
	NOfAppointments = 3
End Enum

Public Class ResourcesSortingHelper
    Public Shared Function GetItems() As IEnumerable
        Return TryCast(New List(Of ResourceSortType)() From { _
            New ResourceSortType() With { _
                .ID = CInt(ResourcesSortOrder.None), _
                .Type = "None" _
            }, _
            New ResourceSortType() With { _
                .ID = CInt(ResourcesSortOrder.Ascending), _
                .Type = "A-Z Order" _
            }, _
            New ResourceSortType() With { _
                .ID = CInt(ResourcesSortOrder.Descending), _
                .Type = "Z-A Order" _
            }, _
            New ResourceSortType() With { _
                .ID = CInt(ResourcesSortOrder.NOfAppointments), _
                .Type = "N-Appointments" _
            } _
        }, IEnumerable)
    End Function
End Class